content/03.results.md

```diff
@@ -32,8 +32,9 @@ Performance of models across four classification tasks. In each panel the loess
 ](./images/full_signal_combined.svg ){#fig:prediction_combined}
 
 ### There is predictive non-linear signal in transcriptomic data
-Our results to this point are consistent with a model where the predictive signal present in transcriptomic data is entirely linear.
+Our results to this point are consistent with a world where the predictive signal present in transcriptomic data is entirely linear.
 If that were the case, non-linear models like neural networks would fail to give any substantial advantage.
+However, based on past results we expect there to be relevant nonlinear biological signal [CITE clustermatch].
 To get a clearer idea of what that would look like, we simulated three datasets to better understand model performance for a variety of data generating processes.
 We created data with both linear and non-linear signal by generating two types of features: half of the features with a linear dividing line between the simulated classes and half with a non-linear dividing line (see [Methods](#methods) for more detail).
```

Comment:
Re. "dividing line", if it were me I think I'd probably go with decision boundary. It's a pretty widely used term in the ML/stats literature, and I assume the main audience for this paper will be people with at least some computational background.

Up to you, though, I think what you have is reasonably clear too.

Reply:
Although on second thought, I think you could probably avoid using "dividing line" here: you could say something like

> We created data with both linear and non-linear signal by generating two types of features: half of the features are linearly separable by simulated class, and half are not linearly separable (see [Methods](#methods) for more detail).

Maybe that could be written more clearly but hopefully you get the idea. Not sure about the other places you use the term, though - it may be harder to remove it entirely.

Reply:
I think I'll end up going with "decision boundary". It's more easily understandable than decision surface, and much less unpleasant to read than "linear dividing line"

Reply:
Right, that's true!